content/blog/esc-versioning-launch/index.md

```diff
@@ -0,0 +1,102 @@
+---
+title: "Unveiling Pulumi ESC Versioning: Manage Secrets and Deployments with Confidence"
+allow_long_title: true
+date: 2024-06-05T00:00:00-07:00
+draft: false
+meta_desc: "The new ESC Versioning features allows you bring agile practices to your secrets and configuration management, and streamline deployments"
+meta_image: "meta.png"
+authors:
+  - arun-loganathan
+  - pat-gavlin
+tags:
+  - esc
+  - secrets
+  - features
+---
+
+Since the launch of [Pulumi Environments, Secrets and Configuration]((/product/esc)) (ESC), our developer-first configuration and secrets management platform, we've seen exponential growth in usage. Customers have used it to simplify their secrets and configuration management by organizing them into composable collections called 'environments'. Today, we are thrilled to introduce a comprehensive suite of versioning features, giving you unprecedented control and confidence in managing your environments.
+
+<!--more-->
+
+## Uncover the Key Features of Pulumi ESC Versioning
+
+Pulumi ESC Versioning introduces a range of powerful features:
+
+- **Immutable Revision History**: Every time you save an environment, a new, immutable revision is created, preserving a complete and tamper-proof history of your changes.
+- **Compare Revisions Side-by-Side**: Gain a clear understanding of changes between revisions with our detailed side-by-side comparison view. This helps identify differences, troubleshoot issues, and ensure smooth transitions between versions.
+- **Tag Revisions**: Assign meaningful names to specific revisions, enabling you to easily manage and deploy different versions of your environments. You can use tags like `Production`, `v1.2.1`, `Stable`, `Pre-release`, and `In-development` to clearly communicate the purpose or stage of each version. Each environment has a built-in `latest` tag that always points to the environment’s most recent revision.
+- **Import Specific Versions**: Precisely control which version of an environment is imported, either by specifying a tag (e.g., `environmentA@PROD`) or using a specific revision number (e.g., `environmentB@2`). This method also applies when [importing environments](/docs/esc/environments/#importing-other-environments) via [Pulumi Stack Config](/docs/esc/environments/#importing-other-environments) or via other environments, to effectively 'pin' the desired version. 
```

Comment:
Would love if we highlighted this more, as this is IMO the key win.  Concepts like "control blast radius" or "controlled rollout" to have people see the value.